src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.vb

```diff
@@ -489,18 +489,20 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
             If _unconfiguredProject Is Nothing OrElse _configuredProject Is Nothing Then
                 GetProjectsAndProvider()
             End If
-            'The TextBox needs to have the relative path, so the property isn't linked to the TextBox. It must be set manually.
-            AddOrChangeLicenseItem(_previousLicenseFileName, relativeFilePath)
-            LicenseFileNameTextBox.Text = relativeFilePath
-            _previousLicenseFileName = LicenseFileNameTextBox.Text
-            'If the user has changed the directory on their PackageLicenseFile, we should keep it there
-            Dim PackageLicenseFileSet = TryCast(TryGetNonCommonPropertyValue(GetPropertyDescriptor("PackageLicenseFile")), String)
-            If PackageLicenseFileSet IsNot Nothing AndAlso PackageLicenseFileSet IsNot "" Then
-                Dim currentPackageLicenseFileDirectory = Path.GetDirectoryName(PackageLicenseFileSet)
-                Dim potentialFullString = Path.Combine(currentPackageLicenseFileDirectory, Path.GetFileName(relativeFilePath))
-                SetCommonPropertyValue(GetPropertyDescriptor("PackageLicenseFile"), potentialFullString)
-            Else
-                SetCommonPropertyValue(GetPropertyDescriptor("PackageLicenseFile"), Path.GetFileName(relativeFilePath))
+            If relativeFilePath = String.Concat(relativeFilePath.Split(Path.GetInvalidFileNameChars())) Then
```

Comment:
Does this work for a path like `Foo\Bar\Baz.cs` or `Foo/Bar/Baz.cs`? The slashes are invalid file name characters, but are valid in paths.

Reply:
No - that was a mistake from testing. It should have been InvalidPathChars.

Reply:
Can you use [`String.IndexOfAny`](https://docs.microsoft.com/en-us/dotnet/api/system.string.indexofany?view=netframework-4.8#System_String_IndexOfAny_System_Char___) instead?

Reply:
Sure - Would that be significantly faster?

Reply:
It would avoid extra string allocations, but mostly it'd be clearer.